docs/_static/css/custom.css

```diff
@@ -71,3 +71,9 @@ img:not(.logo__image, .footer_logo) {
 .footer_logo {
 	max-height: 60px;
 }
+
+.table table, caption,thead,tbody, th, td{
```

Comment:
I wouldn't advise applying this to *all* tables. I think if someone wants to add a border to all table cells, they can add a `border` class to the table directive. This can be modified to only apply the styling to tables with that class. 

Using the example from pydata sphinx theme, I added the argument `:class: border` to the directive. This will add the class ` border` to the table.

```
.. list-table:: List tables can have captions like this one.
    :widths: 10 5 10 50
    :class: borders
    :header-rows: 1
    :stub-columns: 1

    * - List table
      - Header 1
      - Header 2
      - Header 3 long. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam sit amet mauris arcu.
    * - Stub Row 1
      - Row 1
      - Column 2
      - Column 3 long. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam sit amet mauris arcu.
    * - Stub Row 2
      - Row 2
      - Column 2
      - Column 3 long. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam sit amet mauris arcu.
    * - Stub Row 3
      - Row 3
      - Column 2
      - Column 3 long. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam sit amet mauris arcu.
```

Reply:
Good to know! Yes, it makes sense to leave the option to who is using the theme. I'll let him know about the :class: border directive.  We can disregard this PR